uiowa/uiowa
# Fixed publisher permissions

# How to test
Make sure I added back publisher permissions I removed previously: https://github.com/uiowa/uiowa/pull/3663/files
